runtime/src/bank.rs

```diff
@@ -6928,6 +6931,27 @@ impl Bank {
         self.hashes_per_tick = Some(hashes_per_tick);
     }
 
+    fn update_block_limits(&self) {
+        let mut cost_tracker = self.write_cost_tracker().unwrap();
+
+        // For tests and benches, if the limit is set to maximum then skip
+        // setting feature-specific limits.
+        #[cfg(feature = "dev-context-only-utils")]
```

Comment:
code comment explains reason for this. Adding comment on review to draw attention.